info.yml

```diff
@@ -80,6 +86,6 @@ requires_engine_version:
 
 # the frameworks required to run this app
 frameworks:
-    - {"name": "tk-framework-shotgunutils", "version": "v4.x.x"}
+    - {"name": "tk-framework-shotgunutils", "version": "v5.x.x"}
```

Comment:
we already use v5 for the loader. do you see any increased risk in introducing in here? i guess we don't really have a choice if we want to use the hierarchy model...